lib/tic_tac_toe_web/router.ex

```diff
@@ -17,12 +17,12 @@ defmodule TicTacToeWeb.Router do
     pipe_through :browser
 
     get "/", PageController, :index
-    get "/boardstatus", BoardController, :board_status
+    get "/newgame", BoardController, :new_game
```

Comment:
Why not have underscores in the route names?

Also, GameController -> new feels more natural than asking the board for a new game